indy_node/test/request_handlers/conftest.py

```diff
@@ -45,6 +51,83 @@ def schema_request():
                               }})
 
 
+@pytest.fixture(scope="module")
+def rs_schema_handler(db_manager, write_auth_req_validator):
+    return RsSchemaHandler(db_manager, write_auth_req_validator)
+
+
+@pytest.fixture(scope="function")
+def rs_schema_request():
+    authors_did, name, version, _type = "2hoqvcwupRTUNkXn6ArYzs", randomString(), "1.1", "8"
+    _id = authors_did + ':' + _type + ':' + name + ':' + version
+    return Request(identifier=authors_did,
+                   reqId=random.randint(1, 10000000000000000000),
```

Comment:
You already created a function for reqId generation in ` indy_node/test/api/helper.py`. I think it can be used here.

Reply:
I agree I will add code to use the generator from helper.py